src/main/com/algos/src/maths/AbsoluteMax.java

```diff
@@ -13,12 +13,12 @@ public static int getMaxValue(int... numbers) {
         if (numbers == null || numbers.length == 0) {
             throw new IllegalArgumentException("Numbers array cannot be empty or null");
         }
-        int absMax = numbers[0];
+        int absMax = 0;  
         for (int i = 1; i < numbers.length; i++) {
-            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
+            if (numbers[i] > absMax) {
```

Comment:
**Performance Issue**: The change from initializing 'absMax' with the first element of the array to zero and the comparison logic from 'Math.abs(numbers[i]) > Math.abs(absMax)' to 'numbers[i] > absMax' may lead to incorrect results and does not correctly find the absolute maximum value. <br> **Fix**: Initialize 'absMax' with the absolute value of the first element of the array and compare the absolute values within the loop. <br> **Code Suggestion**: 
 ```
 -        int absMax = 0;
 +        int absMax = Math.abs(numbers[0]);
 -            if (numbers[i] > absMax) {
 +            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
 ```